README.md

```diff
@@ -247,30 +247,22 @@ Example:
 'deny_by_default' => false,
 ```
 
-> ##### deny_by_default with zf-oauth2
->
-> When using `deny_by_default => true` with > [zf-oauth2](https://github.com/zfcampus/zf-oauth2),
-> you will need to explicitly allow POST on the OAuth2 controller in order for Authentication
-> requests to be made.
-> 
-> As an example:
->
-> ```php
-> `authorization` => array(
->     'deny_by_default' => true,
->     'ZF\\OAuth2\\Controller\\Auth' => array(
->         'actions' => array(
->             'token' => array(
->                 'GET'    => false,
->                 'POST'   => true,   // <-----
->                 'PATCH'  => false,
->                 'PUT'    => false,
->                 'DELETE' => false,
->             ),
->         ),
->     ),
-> ),
-> ```
+##### deny_by_default with zf-oauth2
+
+When using `deny_by_default => true` with [zf-oauth2](https://github.com/zfcampus/zf-oauth2) you must explicitly allow POST on the OAuth2 controller in order for Authentication requests to be made.
+ 
+```php
+'authorization' => array(
+    'deny_by_default' => true,
+    'ZF\\OAuth2\\Controller\\Auth' => array(
+        'actions' => array(
+            'token' => array(
+                'POST'   => false,   // <-- explictly exclude authentication
```

Comment:
The verbiage and the example do not agree.

Verbiage:

```
you must explicitly allow POST on the OAuth2 controller in order for Authentication requests to be made.
```

code:

```php
'POST' => false,
```

You claim the documentation was incorrect, but now you're making it explicitly contradictory... Can you verify which is correct? (POST should be allowed? or POST should be excluded?).

Thanks!